view/next-project/src/components/common/Textarea/Textarea.tsx

```diff
@@ -6,11 +6,13 @@ import s from './Textarea.module.css';
 interface Props {
   className?: string;
   placeholder?: string;
+  id?: string;
   value?: string | number;
   onChange?: (
     e: React.ChangeEvent<HTMLTextAreaElement> | React.ChangeEvent<HTMLInputElement>,
   ) => void;
   children?: React.ReactNode;
+  type?: string;
```

Comment:
propsのidとtypeの追加ありがとう！
ただ、textareaタグはtypeを持たないので、idだけで大丈夫
あと実際のtextarea（23行目）にもidとtypeを渡してあげないと機能しないので注意！
参照：[textareaタグのドキュメント](https://developer.mozilla.org/ja/docs/Web/HTML/Element/textarea)

```suggestion
  className?: string;
  placeholder?: string;
  id?: string;
  value?: string | number;
  onChange?: (
    e: React.ChangeEvent<HTMLTextAreaElement> | React.ChangeEvent<HTMLInputElement>,
  ) => void;
  children?: React.ReactNode;
```

```
    <textarea
      id={props.id} // idをtextareaに与える必要がある
      className={clsx(s.textarea, className)}
      placeholder={props.placeholder}
      value={props.value}
      onChange={props.onChange}
    >
```